forge-gui/res/cardsfolder/upcoming/final_word_phantom.txt

```diff
@@ -0,0 +1,8 @@
+Name:Final-Word Phantom
+ManaCost:2 U
+Types:Creature Spirit Detective
+PT:1/4
+K:Flash
+K:Flying
+S:Mode$ CastWithFlash | ActivationPhases$ End of Turn | ValidCard$ Card | ValidSA$ Spell | Caster$ You | Description$ During each opponent's end step, you may cast spells as though they had flash
```

Comment:
`ActivationPhases$ End of Turn`  isn't correct. Any ideas what would be correct here ?

Reply:
just Phases$?
you're not activating anything

Reply:
`PlayerTurn$ Opponent`